packages/block-editor/src/components/color-palette/test/control.js

```diff
@@ -10,9 +10,22 @@ import ColorPaletteControl from '../control';
 
 const noop = () => {};
 
+async function renderAndValidate( ...renderArgs ) {
```

Comment:
Not sure what we ended up deciding re. the `render` function — for now, I'm using a composite-specific workaround, but I'm not sure how well this would scale

Reply:
Looks good for this particular instance. Since this is a testing detail, let's worry about it if it actually needs to scale.